client/src/store/musicPlayerSlice.ts

```diff
@@ -235,14 +213,14 @@ export const musicPlayerSlice = createSlice({
   },
 });
 
-export const {
-  setPlaylist,
-  setSelectedPlaylist,
-  setPlaylistSongs,
-  setCurrentTrack,
-  setMusicIsPlaying,
-  setShuffleState,
-  setRepeatState,
-} = musicPlayerSlice.actions;
+// export const {
+//   setPlaylist,
+//   setSelectedPlaylist,
+//   setPlaylistSongs,
+//   setCurrentTrack,
+//   setMusicIsPlaying,
+//   setShuffleState,
+//   setRepeatState,
```

Comment:
eventually try to remove unnecessary commented code, i like how the code is more readable in your project.